johnzon-mapper/src/test/java/org/superbiz/ExtendMappingTest.java

```diff
@@ -62,9 +62,9 @@ public MyMappings() {
                     -1, true, true, true, false, false, false,
                     new FieldAccessMode(false, false),
                     StandardCharsets.UTF_8, String::compareTo, false, null, false, false,
-                    emptyMap(), true, false, true,
-                    null, null, null, null, null,
-                    type -> new EnumConverter(type)));
+                    emptyMap(), true, false, true, null,
```

Comment:
see previous comment but overall this test is a contract so any diff is likely a bug